### PR TITLE
polish: cms proto field backward-compatible

### PR DESCRIPTION
**Pull request by sfwn**
polish field name backward-compatible.

### CODE REVIEW

File: core/pipeline/cms/cms.proto [hunk 1]
```diff
@@ -120,30 +120,30 @@ message CmsListNsResponse {
...
 }
 
 message CmsNsConfigsDeleteRequest {
   string ns = 1;
-  string pipeline_source = 2 [(validator.field) = {string_not_empty: true}];
-  bool delete_ns = 3 [json_name="deleteNS"];
-  bool delete_force = 4;
-  repeated string delete_keys = 5;
+  string pipelineSource = 2 [(validator.field) = {string_not_empty: true}];
+  bool deleteNs = 3 [json_name="deleteNS"];
```
Comment (Effet) [inline core/pipeline/cms/cms.proto:135]: why not change json name

**Review by Effet — CHANGES_REQUESTED** [review]

File: core/pipeline/cms/cms.proto (hunk 1 above)
Comment (sfwn) [reply core/pipeline/cms/cms.proto:135]: json_name for backward-compatible.
use `Ns` because all the other places use this format.

Comment (Effet) [reply core/pipeline/cms/cms.proto:135]: so you should comment it in code

Comment (sfwn) [reply core/pipeline/cms/cms.proto]: Added.

**Review by Effet — APPROVED** [review]

**Pull request merged**